EMIP/1001-2000/EMIP01974.xml

```diff
@@ -48,19 +49,168 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                            <watermark/>
                         </support>
                         <extent>
-                           <measure unit="leaf"/>
+                           <measure unit="leaf">iv + 184</measure>
```

Comment:
measure should contain only the number, not the letter and signs. https://betamasaheft.eu/Guidelines/?id=objectDescription 
this applies to all entries